app/src/main/java/org/wikipedia/history/db/HistoryEntryWithImageDao.kt

```diff
@@ -82,6 +82,10 @@ interface HistoryEntryWithImageDao {
             entryWithImage.displayTitle, 0, entryWithImage.namespace, entryWithImage.timestamp,
             entryWithImage.source, entryWithImage.timeSpentSec)
         entry.title.thumbUrl = entryWithImage.imageName
+
+        val hasDescription = entryWithImage.description?.isNotEmpty() ?: false
+        entry.title.description = if (hasDescription) entryWithImage.description else null
```

Comment:
You can simply assign the value instead.

```
entry.title.description = entryWithImage.description
```

Reply:
Ok, I wasn't sure if it was a problem :)